examples/tracetest-synthetic-monitoring/tracetest/tracetest.config.yaml

```diff
@@ -0,0 +1,21 @@
+postgres:
+  host: postgres
+  user: postgres
+  password: postgres
+  port: 5432
+  dbname: postgres
+  params: sslmode=disable
+
+telemetry:
+  exporters:
+    collector:
+      serviceName: tracetest
+      sampling: 100 # 100%
+      exporter:
+        type: collector
+        collector:
+          endpoint: otel-collector:4317
+
+server:
+  telemetry:
+    exporter: collector
```

Comment:
I think this needs the `applicationExporter` as well to work. I had this issue once and it did not work properly without it.